pkg/ebpf/tcp_queue_length.go

```diff
@@ -29,7 +29,7 @@ type TCPQueueLengthTracer struct {
 }
 
 func NewTCPQueueLengthTracer() (*TCPQueueLengthTracer, error) {
-	source_raw, err := bytecode.Asset("tcp-queue-length-kern.c")
+	source_raw, err := bytecode.Asset("pkg/ebpf/c/tracer-ebpf-debug.o")
```

Comment:
Just wondering why this changed from tcp-queue-length-kern.c. Do we not use this code anymore, or was it incorporated with the tracer-ebpf-debug code?

Reply:
Yikes, this should be the path to tcq-queue-length-kern.c -- nice catch!